### DISCUSSION TITLE
Prisma excute postgres functions inside select

### DISCUSSION THREAD

**Discussion by Siddhesh1992** [prisma/prisma#13858] [Q&A] [answered]
I want to execute a PostgreSQL function 
TO_CHAR(date_of_implementation,'yyyy-Mon-dd') 
inside select query and there are several joins just to this I have  to execute raw query

**Comment by austincrim** [comment, marked as answer]
Hey @Siddhesh1992 👋,

Yes, if you need execute Postgres functions, you will need to use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
